android/build.gradle

```diff
@@ -65,12 +65,13 @@ preBuild.dependsOn createDataVersion
 
 
 android {
-    compileSdkVersion 33
+    compileSdk 33
     ndkVersion "25.1.8937393"
+    namespace "com.reecedunn.espeak"
```

Comment:
just curious what it's for?

Reply:
> Starting with AGP 7.3.0-alpha04, if you use Gradle to build your project, AGP generates a warning if you use the package attribute in the manifest file. To set the namespace for your app, use the namespace property in the module-level build.gradle file. To learn more, see [Set a namespace](https://developer.android.com/studio/build/configure-app-module#set-namespace).

Source: <https://developer.android.com/studio/releases/gradle-plugin#package-deprecated>